packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -302,6 +302,100 @@ Use the `add` operation to add new claims to a token. If you use the `add` opera
   ]
 }
 ```
+
+You can also use the `add` operation to add new members to existing JSON objects and new elements to existing arrays. For example, you have a JSON object in a claim called `employee_profile`, and you want to add the `department_id` member to the claim.
+
+The existing target JSON object:
+```json
+{
+"employee_profile": {
+       "employee_id": "1234",
+       "name": "Anna"
+  }
+}
+```
+Add `department_id` by specifying the claim in the path, followed by the name of the object member.
+
+> Note: Attempting to add a member within a JSON object that doesn't exist or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.
```

Comment:
same as for https://github.com/okta/okta-developer-docs/pull/493#discussion_r296942600